### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by silvano-a**

### CODE REVIEW

**Review by NiekKeijzer — COMMENTED** [review]
This could be simplified by using a [version matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) 

Something like this should work: 
```yaml
jobs: 
  php-tests:
    name: Suite (PHP ${{ matrix.php-versions }} on ${{ matrix.operating-system }})
    runs-on: ${{ matrix.operating-system }}
    strategy:
      fail-fast: true
      matrix:
        operating-system:
          - ubuntu-22.04
        php-versions:
          - "8.1"
          - "8.0"
          - "7.4"
    steps: 
      - name: Checkout
          uses: actions/checkout@v2

      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php-versions }}

      - name: Install Dependencies
        run: |
          composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress
      - name: Execute tests (Unit tests) via PHPUnit
        run: |
          vendor/bin/phpstan
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        with:
+          php-version: '8.1'
+
+      - name: Install Dependencies
+        run: |
+          composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress
+
+      - name: Execute tests (Unit tests) via PHPUnit
+        run: |
+          vendor/bin/phpstan
```
Comment (bonroyage) [inline .github/workflows/ci.yml:23]: `name` doesn't match the action being run

**Review by bonroyage — APPROVED** [review]
Checking if the action will run on approval

**Comment by silvano-a** [comment]
https://github.com/silvano-a/laravel-autodiscovery-lock/actions/runs/3572807316

**Review by NiekKeijzer — APPROVED** [review]

**Pull request merged**